.github/workflows/uffizzi-build.yml

```diff
@@ -0,0 +1,86 @@
+name: Build PR Image
+on:
+  pull_request:
+    types: [opened,synchronize,reopened,closed]
+
+jobs:
+
+  build-appication:
+    name: Build and Push `f-ui-kit`
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name != 'pull_request' || github.event.action != 'closed' }}
+    outputs:
+      tags: ${{ steps.meta.outputs.tags }}
+    steps:
+      - name: Checkout git repo
+        uses: actions/checkout@v3
+      - name: Generate UUID image name
+        id: uuid
+        run: echo "::set-output name=uuid::$(uuidgen)"
```

Comment:
We're trying to deprecate `set-output`. the sooner the better.  Only change it here if this PR is likely to be accepted.  This is a good example https://github.com/UffizziCloud/quickstart/commit/3731ddb33771ddda88d93db2356dddf48f1ae7b6

We can also upgrade the third-party actions we're using like in this commit https://github.com/UffizziCloud/quickstart/commit/af17b37b9513f28e6299577611947ec8fe3cdade

Reply:
Done.

Reply:
As we agreed - we will apply this change later.